src/development/loadbalancer/nginx.conf

```diff
@@ -43,6 +47,14 @@ http {
 			  proxy_pass		      http://localtest/Home/;
 		}
 
+    location /receipt/ {
+			  proxy_pass		      http://receiptcomp/receipt/;
```

Comment:
```suggestion
			  proxy_pass		      http://receiptcomp/receipt/;
			  error_page 502 /502Receipt.html;
```

Reply:
Would also require
```
    location /502Receipt.html {
      root /www;
    }
```
And also the creation of a custom error page with info about the receipt component.

Reply:
@ivarne  thanks. I was not aware of this concept. I have added error page

Reply:
Great!